src/server/api/controllers/voyages.controller.js

```diff
@@ -0,0 +1,37 @@
+"use strict";
+
+const HttpError = require("../lib/utils/http-error");
+const knex = require("../../config/db");
+
+// to get all the voyages
+const getVoyages = () => {
+  return knex("voyages").select("*");
+};
+
+//to create a voyage
+const createVoyage = async ({ body }) => {
+  try {
+    return knex("voyages").insert({
+      vessel_id: body.vessel_id,
```

Comment:
You could destructure it like, but just an option 🙌 
```
const { vessel_id, ... } = body;
...
return knex("voyages").insert({
  vessel_id,
   ...
```

Reply:
I will try to keep it in my mind for next time :)